way_to_home/user_profile/models.py

```diff
@@ -42,3 +42,27 @@ def create(cls, user, first_name='', last_name='', telegram_id=None):  # pylint:
             return user_profile
         except (ValueError, IntegrityError, OperationalError) as err:
             LOGGER.error(f'Unsuccessful user profile creating. {err}')
+
+    def update(self, first_name=None, last_name=None, telegram_id=''): # pylint: disable=arguments-differ
+        """Method for updating of user profile object"""
+        with transaction.atomic():
+            if first_name:
+                self.first_name = first_name
+            if last_name:
+                self.last_name = last_name
+            if telegram_id != '':
+                self.telegram_id = telegram_id
+            try:
+                self.save()
+                return True
+            except (ValueError, OperationalError) as err:
+                LOGGER.error(f'Unsuccessful profile\' parameters updating with id={self.id}. {err}')
+                return False
+
+    @classmethod
+    def find_by_telegram_id(cls, telegram_id):
```

Comment:
As I wrote above, please change it to `get_by_telegram_id`

Reply:
as i wrote above, done)